scripts/modules/calendarFunctions.js

```diff
@@ -0,0 +1,86 @@
+import { createTextEl } from "./generalFunctions.js";
+
+///////
+//// Functions for calendar
+
+export function generateCalendar(currentMonth, currentYear, calEle) {
+  calEle.innerHTML = "";
+
+  ////////
+  //// display current month & year
```

Comment:
Think if this could be broken down into two functions, for example a function that returns an array of days for the given month, a functon that creates dom elements for each day